src/Footer/SubmitButton/SubmitButton.js

```diff
@@ -28,22 +28,26 @@ const SubmitButton = ({
   if (hide) return null
 
   const canSubmit = () => {
-    const fileIds = []
-    fields.map((field) => {
-      if (field.type === 'file') {
-        fileIds.push(field.id)
-      }
-    })
-
-    for (let i in fileIds) {
-      if (values[fileIds] && !values[fileIds][i]['responce']) {
-        return false
+    let isDisabled = true;
+    for (let i = 0; i < fields.length; i++){
+      if (fields[i].type === 'file') {
+        const filesArray = values[fields[i].id];  
+        if (filesArray) {
+          for (let j = 0; j < filesArray.length; j++){
+            let { percentage } = filesArray[j];
+            if (percentage < 100 && !filesArray[j]['responce']) {  
+              isDisabled = false;
```

Comment:
Just return false if the file isn't uploaded.
And in the end of function just return true.